README.md

```diff
@@ -186,9 +186,9 @@ Rules are applied in order. In other words, the second Rule can override values
 
 Platform requirements:
 
-* [PHP] 7.4 or later (tested up to 8.0)
-* [MediaWiki] 1.35 or later (tested up to 1.37)
-* [Wikibase] 1.35 or later (tested up to 1.37)
+* [PHP] 7.4 or later (tested up to 8.3)
```

Comment:
Shouldn't this be 8.4?

Reply:
Technically CI on MW master passes on 8.4 today. I'm slightly unsure if we should add that as official support yet. Or maybe that's good enough.